homeworks/react-basics/yahello_yana/kottans-lectures/src/lecturesDetails.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+
+class LectureDetails extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {isOpened: false};
+    }
+
+    toggleState() {
+        this.setState({isOpened: !this.state.isOpened})
+    }
+
+    render() {
+        let isShow;
```

Comment:
You don't need `isShow` here. You can just check if `this.state.isOpened ` is `true` in return, this should look like: 
```
{this.state.isOpened && <div>...</div>}
```

Reply:
fixed